go.mod

```diff
@@ -142,3 +142,5 @@ replace k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.0.0-20
 replace k8s.io/node-api => k8s.io/node-api v0.0.0-20190918163711-2299658ad911
 
 replace github.com/uber-go/atomic => go.uber.org/atomic v1.5.0
+
+replace github.com/pingcap/advanced-statefulset => /data/go/src/github.com/pingcap/advanced-statefulset
```

Comment:
probably don't want to commit this

Reply:
this is for local development only, I'll remove this